src/ontology/edits/oeo-physical.omn

```diff
@@ -415,6 +415,120 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1016",
+        rdfs:label "diesel fuel role"@en
+    
+    SubClassOf: 
+        OEO_00000001
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010241>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gasoline fuel is a combustion fuel that has a gasoline fuel role.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/960
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1027",
+        rdfs:label "gasoline fuel"@en
+    
+    EquivalentTo: 
+        OEO_00000099
+         and (<http://purl.obolibrary.org/obo/RO_0000087> some <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010239>)
+    
+    SubClassOf: 
+        OEO_00000099
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010242>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A diesel fuel is a combustion fuel that has a diesel fuel role.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/960
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1027",
+        rdfs:label "diesel fuel"@en
+    
+    EquivalentTo: 
+        OEO_00000099
+         and (<http://purl.obolibrary.org/obo/RO_0000087> some <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010240>)
+    
+    SubClassOf: 
+        OEO_00000099
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010243>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gasoline engine is an internal combustion engine that uses a gasoline fuel.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/960
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1027",
+        rdfs:label "gasoline engine"@en
+    
+    SubClassOf: 
+        OEO_00010029,
+        OEO_00000503 some <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010241>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010244>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A diesel engine is an internal combustion engine that uses a diesel fuel.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/960
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1027",
+        rdfs:label "diesel engine"@en
+    
+    SubClassOf: 
+        OEO_00010029,
+        OEO_00000503 some OEO_00000131
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-physical/OEO_00010245>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gasoline vehicle is an internal combustion vehicle that has only a gasoline engine as motor.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A gasoline vehicle is an internal combustion vehicle that has only a gasoline engine as motor for traction.",
```
![grafik](https://user-images.githubusercontent.com/38690039/154013402-8340bb15-b4a7-42a1-84a2-b09e39510e42.png)
Since electric motors can be used for other stuff than traction, also in internal combustion vehicles, we once introduced the class `traction motor`. Maybe we should structure it the other way round and have it as parent class for all kinds of traction motors / engines?!

Reply:
I am okay with extending the definition here.

`traction motor` in the current definition (_A traction motor is an electric motor used for propulsion._) was introduced to properly axiomatise e.g. `electric vehicle` and `plug-in hybrid vehicle`:

```
electric vehicle
    has part some traction motor
    ...

plug-in hybrid electric vehicle
    has part some internal combustion engine
    has part some traction motor
    ...
```

Even if we introduce a (general) traction motor class, we still need the (electric) traction motor class to describe e.g. electric vehicle. The general traction motor class is probably an equivalent class to `motor and participates in some traction process` or something like that.

But this is far outside the scope of this PR. So I open an separate issue.

Reply:
I added now _for propulsion_ instead of _for traction_, as we used that in a couple of existing vehicle and motor classes. Additionally, I added now also `diesel motor` and `gasoline motor` as alternative terms to `diesel engine` and `gasoline engine`.